markdown/dev/guides/markdown/frequent-mistakes/en.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Frequent mistakes
+order: 100
+---
+
+Some things to keep in mind when working in Markdown are:
+
+- There is no need to add a *glossary* section to documentation. We use a plugin called [remark-jargon](https://github.com/freesewing/freesewing/blob/develop/packages/remark-jargon/README.md) to explain terms. Information can be found at the link.
+
```

Comment:
I think something is going wrong because of the blank lines between each bullet line (lines 9 and 11 for example.

This is what it looks like for me:
![image](https://user-images.githubusercontent.com/9117775/152604004-8ac841c5-b7d6-4ca8-b992-900366810dec.png)

Removing the lines resolves the issue.

Reply:
Huh, it didn't do that in the preview! I'll fix it :)

Reply:
Not sure what is going on. Some other parts of the page are also being rendered strangely on my local freesewing.dev but they look fine in GitHub: https://github.com/freesewing/freesewing/blob/1c8afb2100e58c4cde71b652975b208c847ab955/markdown/dev/guides/markdown/frequent-mistakes/en.md

Will play around with this tomorrow.